apps/healthlocker/mix.exs

```diff
@@ -58,6 +59,7 @@ defmodule Healthlocker.Mixfile do
      {:postgrex, ">= 0.0.0"},
      {:phoenix_html, "~> 2.6"},
      {:phoenix_live_reload, "~> 1.0", only: :dev},
+     {:phoenix_html_sanitizer, "~> 1.0.0"},
```

Comment:
_Assuming_ the addition of `phoenix_html_sanitizer` is to prevent XSS?
https://github.com/elixirstatus/phoenix_html_sanitizer 
no comment/discussion in the issue of why _this_ package was selected ... :thought_balloon:

Reply:
Yeah that is right. I went with this as it is used in HL and I knew roughly how it would work. 
There are a few comments discussing it in HL...
https://github.com/healthlocker/healthlocker/issues/1074
https://github.com/healthlocker/healthlocker/issues/1116
https://github.com/healthlocker/healthlocker/commit/f9c1e648671488e58de16d431fb2cc4a463daf0b